pkgs/development/libraries/kde-frameworks/plasma-framework.nix

```diff
@@ -17,4 +17,15 @@ mkDerivation {
     qtquickcontrols2
   ];
   propagatedBuildInputs = [ kpackage kservice qtbase ];
+  patches = [
+    # patches included in 5.91, fix activity panel pushing windows in wayland
+    (fetchpatch {
+      url = "https://invent.kde.org/frameworks/plasma-framework/-/commit/d40d36057a0ee9fcb4badc7ff8b56844da79dfc8.diff";
+      sha256 = "09dmzx7dmcjjbw4gm3hrxbmkmqx614h6kznhvbfr9vcw88ymc6a9";
```

Comment:
I'm very new to Nix/NixOS, so it's very possible I'm doing something wrong, but I think you may have missed a step when generating this particular hash. This one is correct for the file at the link, but according to `nixos-rebuild switch` isn't correct for the patch itself after processing.
In case this problem isn't on my end, the hash it says it got is `sha256-44MUzHJmS08LlWIkJCad+cvXzWicuy9DqGiYEYA0IJM=`

Reply:
You can't generate the hash with tooling. You need to plump in a wrong one into fetchpatch and then try to build it once.